l10n_br_account_payment_order/models/bank_payment_line.py

```diff
@@ -124,6 +111,32 @@ def default_get(self, fields_list):
              'pagamento nas Ordens de Pagamento',
     )
 
+    payment_mode_id = fields.Many2one(
+        comodel_name='account.payment.mode',
+        string='Payment Mode',
+        ondelete='set null',
+        domain="[('id', 'in', payment_mode_ids)]",
+    )
+
+    payment_mode_ids = fields.Many2many(
```

Comment:
Qual o motivo de ter esse campo aqui ? Já que uma linha só poderá ser associada a um Payment Mode, e caso seja preciso essa informação pode ser obtida pelo campo order_id

Reply:
Por causa da usabilidade. Eu não consegui uma forma de ajustar o domínio dinamicamente pelos modos de pagamento selecionados no order_id. Quando coloquei no onchange o domínio dinâmico não funcionou, e quando coloquei "[('id', 'in', order_id.payment_mode_ids)]" deu um erro de javascript. Com o campo related readonly foi o único jeito que funcionou.

Reply:
No caso do account.payment.line pode existir a questão de se cadastrar uma linha diretamente na tela, mas no caso do objeto bank.payment.line por ser criado automaticamente existe alguma necessidade de existir aqui ?